src/resource_clients/actor.ts

```diff
@@ -268,9 +268,7 @@ export interface ActorExampleRunInput {
     contentType: string;
 }
 
-export interface ActorTaggedBuilds {
-    latest: ActorTaggedBuild;
-}
+export type ActorTaggedBuilds = Record<string, ActorTaggedBuild>
```

Comment:
not sure why this is not checked via eslint, there should be a semicolon at the end 🙃